content/blog/monthly-january-2020.md

```diff
@@ -0,0 +1,68 @@
+---
+title: This Month in TiKV - January 2020
+date: 2020-02-18
+author: Calvin Weng
+---
+
+<!-- Fill in the below from the API: https://developer.github.com/v4/explorer/ -->
+
+Welcome to the first monthly wrap-up for 2020. As folks in China are fighting against the outbreak of coronavirus, we keep our fingers crossed and hope this could end soon. In the meantime, we stay healthy and focus on driving forward our project.
```

Comment:
```suggestion
Welcome to the first monthly wrap-up of 2020. As folks in China are fighting against the outbreak of coronavirus, we keep our fingers crossed and hope this could end soon. In the meantime, we stay healthy and focus on driving forward our project.
```